docs/pom.xml

```diff
@@ -100,6 +100,11 @@
                                     <include>quarkus-azure-app-configuration.adoc</include>
                                     <filtering>false</filtering>
                                 </resource>
+                                <resource>
+                                    <directory>${project.basedir}/../target/asciidoc/generated/config/</directory>
+                                    <include>quarkus-azure-keyvault.adoc</include>
+                                    <filtering>false</filtering>
+                                </resource>
                                 <resource>
```

Comment:
Note: this comment is not for changes added here, but for other places in this file:
* Shall we also add keyvault which is similar to https://github.com/quarkiverse/quarkus-azure-services/blob/main/docs/pom.xml#L17-L40?

Reply:
See https://github.com/quarkiverse/quarkus-azure-services/pull/218/commits/1b0f9b4ce9d8e8cfba02237c818ba1bdb4c90574.

Reply:
See https://github.com/quarkiverse/quarkus-azure-services/issues/223.